src/sentry/incidents/endpoints/validators.py

```diff
@@ -92,8 +98,8 @@ class MetricAlertComparisonConditionValidator(NumericComparisonConditionValidato
 
 
 class MetricAlertsDetectorValidator(BaseGroupTypeDetectorValidator):
-    data_source = SnubaQueryDataSourceValidator(required=True)
-    data_conditions = MetricAlertComparisonConditionValidator(many=True)
+    data_source = SnubaQueryDataSourceValidator(required=True, many=True)
+    data_conditions = MetricAlertComparisonConditionValidator()
```

Comment:
I'm planning on updating the names in a separate PR so they match whether they're singular or plural (many to many)

Reply:
Actually I might be confused about the `data_conditions`, could there still be multiple here?